src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -97,6 +98,17 @@ private static bool FastColorOffsetConvert(Type outputType, ColorOffset input, r
 			return false;
 		}
 
+		private static bool FastColorConvert(Type outputType, Windows.UI.Color color, ref object output)
+		{
+			if (outputType == typeof(SolidColorBrush))
+			{
+				output = new SolidColorBrush(color);
```

Comment:
The fact that this creates a new brush every time means people should be discouraged from actually taking advantage of this capability ;)

Reply:
True... Though it's probably best to avoid sharing issues ?

Reply:
Yeah I think this is the correct way to handle it... we should just counsel people to avoid 'it' in the first place :) For widely-used resources at least.

Reply:
You have a better idea ?